docs/DEVELOPMENT.md

```diff
@@ -0,0 +1,182 @@
+# Development
+
+## Getting Started
+
+1. [Create a GitHub Account](https://github.com/join)
+2. [Setup GitHub access via SSH](https://help.github.com/articles/connecting-to-github-with-ssh/)
+3. [Create and Checkout fork of this repository](#checkout-your-fork)
+4. [Install requirements](#requirements)
+5. [Two ways of running Hub](#two-ways-of-running)
+    - On local machine
+    - On a Kubernetes cluster
+
+### Checkout your fork
+
+To check out this repository:
+
+1. Create your own [fork of this repository](https://help.github.com/articles/fork-a-repo/)
+2. Clone it to your machine:
+```
+git clone git@github.com:${YOUR_GITHUB_USERNAME}/hub.git
+cd hub
+
+git remote add upstream git@github.com:tektoncd/hub.git
+git remote set-url --push upstream no_push
+```
+
+Adding the upstream remote sets you up nicely for regularly [syncing your fork.](https://help.github.com/articles/syncing-a-fork/)
+
+### Requirements
+
+You must install these tools:
+
+1. [`go`](https://golang.org/doc/install): The language hub apis are built in.
+2. [`git`](https://help.github.com/articles/set-up-git/): For source control
+
+You may need to install more tools depending on the way you want to run the hub.
+
+### Two ways of Running
+
+- **On local machine**
+
+    For running on local machine you need postgresql database.
+
+    Two ways to run postgresql database:
```

Comment:
Here can we add if user has postgres installed then the user needs to create a database ?

Reply:
yeah